patterns/atoms/buttons/button.php

```diff
@@ -0,0 +1,13 @@
+<?php
+$args = wp_parse_args($args, [
+	'label' => '',
+	'classes' => [],
+] );
+
+$classes = ! empty( $args['classes'] ) && is_array( $args['classes'] ) ? implode( " ", $args['classes' ] ) : '';
```

Comment:
1. By casting to `array` we don't have to test for `empty` or if is an `array``
2. By using `trim` on the final result we make sure empty array creates empty string.